src/main/java/com/marianbastiurea/lifeofbees/LifeOfBees.java

```diff
@@ -11,61 +11,77 @@ public class LifeOfBees {
 
     public LifeOfBees() {
         this.apiary = new Apiary();
+        createHives(apiary.getNumberOfHives());
+    }
+
+    @Override
+    public String toString() {
+        return "LifeOfBees{" +
+                "apiary=" + apiary +
+                ", hiveIdCounter=" + hiveIdCounter +
+                '}';
     }
 
     // Method to create 10 hives and store them in the apiary
-    public void createHives( int numberOfStartingHives) {
+    public void createHives(int numberOfStartingHives) {
+
         List<Hive> hives = new ArrayList<>();
-        for (int i = 1; i < numberOfStartingHives+1; i++) {
+        for (int i = 1; i < numberOfStartingHives + 1; i++) {
             Hive hive = createHive();
             hives.add(hive);
         }
         apiary.setHives(hives);
-        System.out.println("First "+numberOfStartingHives+ " are "+hives);
+        apiary.setNumberOfHives(numberOfStartingHives);
+        System.out.println("First " + numberOfStartingHives + " are " + hives);
     }
 
     // Method to create a single hive with all its components
     private Hive createHive() {
+
         Random random = new Random();
         Hive hive = new Hive();
+        hive.setItWasSplit(false);// a hive could be split only once in a year
```

Comment:
Use a constructor with parameters to instantiate Hive instead of using setters.

Reply:
i will add a method hibernate where I will changes many parameters for next year. I will fix it then